src/openvic-simulation/map/State.cpp

```diff
@@ -40,13 +50,25 @@ void State::update_gamestate() {
 		average_militancy += province->get_average_militancy() * province_population;
 
 		pop_type_distribution += province->get_pop_type_distribution();
+
+		max_supported_regiments += province->get_max_supported_regiments();
 	}
 
 	if (total_population > 0) {
 		average_literacy /= total_population;
 		average_consciousness /= total_population;
 		average_militancy /= total_population;
 	}
+
+	// TODO - use actual values when State has factory data
+	const int32_t total_factory_levels_in_state = 0;
+	const int32_t potential_workforce_in_state = 0; // sum of worker pops, regardless of employment
```

Comment:
It's good to see mechanics implemented in code because it raises more questions.
In this case: "Which pops to sum here?"
Turns out the answer is pops with `can_work_factory = yes`
I assumed it would instead use the workforce requested by the actual factories but I was wrong.

I'll update the documentation.